src/main/java/org/apache/maven/report/projectinfo/TeamReport.java

```diff
@@ -226,10 +276,7 @@ private void renderTeamMember(Contributor member, Map<String, Boolean> headersMa
                 Properties properties = member.getProperties();
                 String picUrl = properties.getProperty("picUrl");
                 if (picUrl == null || picUrl.isEmpty()) {
-                    picUrl = getGravatarUrl(member.getEmail());
-                }
-                if (picUrl == null || picUrl.isEmpty()) {
-                    picUrl = getSpacerGravatarUrl();
+                    picUrl = getExternalAvatarUrl(member.getEmail());
```

Comment:
What do you want to do here when getExternalAvatarUrl( fails and throws an exception? Probably just don't call sink.figureGraphics below.

Reply:
I want break a build, because images in report can be missing.

Reply:
when user doesn't have an avatar default one will be returned by gravatar

Reply:
Looks like you can handle the IOException in this method. It doesn't need tonescape this class. Just wrap lines 277 through 285 in a try-catch block. The catch block can be empty or it can fill in a default image.